internal/integrations/alerting/email.go

```diff
@@ -59,3 +59,42 @@ func (t *TenantAlertManager) sendEmailExpiringTokenAlert(tenant *dbsqlc.Tenant,
 		},
 	)
 }
+
+func (t *TenantAlertManager) sendEmailTenantResourceLimitAlert(tenant *dbsqlc.Tenant, emailGroup *repository.TenantAlertEmailGroupForSend, payload *alerttypes.ResourceLimitAlert) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	var subject string
+	var summary string
+	var summary2 string
+
+	if payload.AlertType == string(dbsqlc.TenantResourceLimitAlertTypeAlarm) {
+		subject = fmt.Sprintf("Hatchet Limit Alarm! `%s` resource is at %d%% of its limit (%d/%d)", payload.Resource, payload.Percentage, payload.CurrentValue, payload.LimitValue)
```

Comment:
```suggestion
		subject = fmt.Sprintf("%s has exhausted %d%% of its limit (%d/%d)", payload.Resource, payload.Percentage, payload.CurrentValue, payload.LimitValue)
```

Also, if possible, can we map things like `WORKFLOW_RUN` -> `Workflow Run`?

Reply:
added some string transforms